spec/app/domain/authentication/authn-gcp/decoded_token_spec.rb

```diff
@@ -303,13 +316,18 @@ def decoded_token_hash(token_str)
           )
         end
 
-        it "raises an error" do
-          expect { decoded_token }.to raise_error(Errors::Authentication::Jwt::TokenClaimNotFoundOrEmpty)
+        it "does not raise an error" do
+          expect { decoded_token }.to_not raise_error
+        end
+
+        it "parses the token expectedly" do
```

Comment:
as expected (it is valid but IMHO not common)

Reply:
It is aligned with the whole file